magento/magento2
# Reports module has no place modifying CMS homepage

### Description

Today I learned (through conversation with a colleague) that the Magento_Reports module has a set-up script which modifies the CMS homepage of Magento. This feels like over-reach. The specific change uses a deprecated feature where each CMS block/page can have layout XML specified in the database. It adds some comments which look like they should be layout instructions, but without any context nor direction for developers.

This pull request removes the modification from future websites. This will have no impact to existing websites as the data patch will have already been applied. I have specifically separated the intended change from the commit where coding standards are applied.

### Manual testing scenarios
1. Install Magento
1. Review layout configuration of `home` CMS page in the database.
   There should be no custom layout configuration set for this page in the database.

### Contribution checklist



### Resolved issues:
1. [x] resolves magento/magento2#37782: Reports module has no place modifying CMS homepage
